### DISCUSSION TITLE
I don't receive an email to reset my password

### DISCUSSION THREAD

**Discussion by SonnenGreif** [community/community#77889] [New to GitHub]
Hello. I tried to change the old password to a new one because I forgot it. To do this, I clicked the “password recovery” button, after which I should have received an email to reset it. But I never received anything (the spam folder is also empty)

**Comment by erickdc7** [comment]
If you're not receiving the password reset email, here are a few steps you can take to troubleshoot the issue:

1. **Check Email Address**:
   Ensure that the email address you entered for the password reset is correct and is the one associated with your account.

2. **Wait for the Email**:
   Sometimes, there can be a delay in the email delivery. Wait for a few minutes and check your inbox again.

3. **Check Spam/Junk Folder Again**:
   Double-check your spam/junk folder to make sure the email didn't end up there.

4. **Check Email Filters**:
   Verify if you have any email filters or rules set up that could be redirecting the email to another folder or deleting it.

5. **Contact Support**:
   If you've tried the above steps and still haven't received the email, contact the support team of the service you're trying to reset the password for. They can assist you further and ensure that the email is sent to you.

6. **Use an Alternative Method**:
   Some services offer alternative methods for account recovery, such as answering security questions or using a mobile phone number. Check if these options are available to you.

Remember to provide the support team with any relevant information, such as your username and the last time you had access to your account, to help them resolve the issue more efficiently.

**Reply by ghost** [reply]
Chatgpt

**Comment by sonlqt** [comment]
Ah yeah, same thing happened me before. Clicked reset, waited, no email, checked spam, nothing. Thought maybe I typed wrong email, but no, still nothing.

Best you try again, wait bit. If still no good, go here and try ask  GitHub support:
https://support.github.com/request

Might work, but eh, best of luck mate
